client/controllers/curatorInbox/curatorSourceDetails.coffee

```diff
@@ -69,16 +69,17 @@ Template.curatorSourceDetails.onRendered ->
           $title.tooltip('hide').attr('data-original-title', '')
         else
           $title.attr('data-original-title', title)
-      @subscribe 'ArticleIncidentReports', source.url
-      if source.enhancements?.dateOfDiagnosis
-        instance.incidentsLoaded.set(true)
-      else
-        Meteor.call 'getArticleEnhancementsAndUpdate', source, (error, enhancements)=>
-          if error
-            notify('error', error.reason)
-          else
-            source.enhancements = enhancements
+      @subscribe 'ArticleIncidentReports', source.url,
+        onReady: ->
```

Comment:
I'm having trouble replicating this. Seems like both the annotated document and the tables will both render after the subscription is ready, so the incidents in either should be consistent?

I did notice a bug with calling the jquery object of the instance before the template is rendered which is fixed in 62ba40d